### ISSUE TITLE
Visibility: Invalid Prop 'offset'

### ISSUE THREAD

**Issue by pflenker** [Semantic-Org/Semantic-UI-React#2116]
I encountered this error in one of my apps, and annoyingly was not able to reproduce it using CodePen. Please let me know if I can do anything to improve this issue!
### Steps
I was not able to recreate this with Codepen. Instead, I followed these steps to create a minimal example:

- Create a new empty app with `create-react-app issuetest`
-  `cd issuetest`, then: install Semantic UI with `yarn add semantic-ui-react --save`
- Change the `App.js` to the following:
```
 import React, { Component } from 'react';
  import {Visibility} from 'semantic-ui-react'
  class App extends Component {
   render() {
     return (
        <Visibility/>
     );
    }
  }
 
 export default App;
```
- `yarn start`

### Expected Result
No warnings.
### Actual Result
Warning: 
> Warning: Failed prop type: Invalid prop `offset` supplied to `Visibility`.
    in Visibility (at App.js:6)
    in App (at index.js:7)

### Version
encountered in 0.73.1, verified in  0.74.0, I did not check earlier versions. 
In case this is needed (since I did not provide a codepen): My dependencies from the package.json:
```
 "dependencies": {
    "react": "^15.6.1",
    "react-dom": "^15.6.1",
    "react-scripts": "1.0.13",
    "semantic-ui-react": "^0.74.0"
  },
```
### Testcase
I was not able to replicate the bug using CodePen, sorry!

**Comment by levithomason** [comment]
Please create a minimal repo in this case.  Unfortunately, we do not have the bandwidth to create projects to debug issues.

Note, the prop type of this is [here](https://github.com/Semantic-Org/Semantic-UI-React/blob/master/src/behaviors/Visibility/Visibility.js#L74).  This will only throw if you supply something other than a string, number, or array of those.

```jsx
    /**
     * Value that context should be adjusted in pixels. Useful for making content appear below content fixed to the
     * page.
     */
    offset: PropTypes.oneOfType([
      PropTypes.number,
      PropTypes.string,
      PropTypes.arrayOf([
        PropTypes.number,
        PropTypes.string,
      ]),
    ]),
```

**Comment by levithomason** [comment]
Just like that, the issue is right where I posted 😄 ^

```diff
offset: PropTypes.oneOfType([
  PropTypes.number,
  PropTypes.string,
- PropTypes.arrayOf([
+ PropTypes.arrayOf(PropTypes.oneOfType([
    PropTypes.number,
    PropTypes.string,
- ]),
+ ])),
]),
```

Patch coming.

**Comment by pflenker** [comment]
Fantastic, thank you!

**Comment by levithomason** [comment]
As a heads up, PropTypes only work in dev.  React doesn't fire them in production.  We also add logic to strip them when minified with dead code elimination.

All that to say, the doc site and the Codepen use the production bundle, hence no propTypes 😉

**Comment by levithomason** [comment]
Fixed in `semantic-ui-react@0.74.1`
